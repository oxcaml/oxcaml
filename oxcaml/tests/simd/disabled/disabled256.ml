
external box_vec256 : int64x4# -> int64x4 = "%box_vec256"
external unbox_vec256 : int64x4 -> int64x4# = "%unbox_vec256"

let f v = box_vec256 (unbox_vec256 v)
