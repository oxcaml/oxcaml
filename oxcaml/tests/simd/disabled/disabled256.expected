Fatal error: exception extension disabled: found 256-bit register whilst emitting camlDisabled256__f_0_1_code, but AVX is not enabled.
